### ISSUE TITLE
idleDisco function

### ISSUE THREAD

**Issue by davidginn524** [Here-Be-Dragons/familamp#3]
I noticed on the idledisco functoin when it does this, the first LED is skipped, the rest of them blink away happily but the first one never starts or changes color. I noticed this when it hit the birthday function

**Comment by Here-Be-Dragons** [comment]
I just ran the code on a strip of pixels on my bench and they appear to be working as expected. Neopixels are fickle things... Can you confirm the first pixel is functional using a basic sketch uploaded to test a pixel, such as:

```c++
#include "neopixel/neopixel.h"

#define PIXEL_COUNT 16
#define PIXEL_PIN D2
#define PIXEL_TYPE WS2812

Adafruit_NeoPixel strip = Adafruit_NeoPixel(PIXEL_COUNT, PIXEL_PIN, PIXEL_TYPE);

void setup() {
    strip.begin();
    strip.show();
}

void loop() {
    strip.setPixelColor(0, 255, 255, 255);
    strip.show();
    delay(1000);
    strip.setPixelColor(0, 255, 0, 0);
    strip.show();
    delay(1000);
    strip.setPixelColor(0, 0, 255, 0);
    strip.show();
    delay(1000);
    strip.setPixelColor(0, 0, 0, 255);
    strip.show();
    delay(1000);
    strip.setPixelColor(0, 0, 0, 0);
    strip.show();
    delay(1000);
}
```

**Comment by davidginn524** [comment]
sorry for the very late reply. while I have not tried this I do know the first pixel is fully working as all of the other functions light the 1st led up

**Comment by Here-Be-Dragons** [comment]
Please look over your `idleDisco()` function.  It should definitely be starting on pixel 0 (the first pixel).  The line that controls which LED is lit is:

```
int randi = random(0,(strip.numPixels() - 1));
```
This is called in a loop that matches the total number of LEDs, though it doesn't not necessarily update each pixel each time.

**Comment by Here-Be-Dragons** [comment]
Fixed in #4
